calico-enterprise/reference/resources/webhook.mdx

```diff
@@ -0,0 +1,146 @@
+---
+description: API for this Calico Enterprise resource.
+---
+
+# Security Event Webhook
+
+A Security Event Webhook (`SecurityEventWebhook`) is a cluster scoped resource that represents instances
+of integrations with external systems through the webhook callback mechanism.
+
+For `kubectl` [commands](https://kubernetes.io/docs/reference/kubectl/overview/), the following case-insensitive aliases
+can be used to specify the resource type on the CLI:
+`securityeventwebhook.projectcalico.org`, `securityeventwebhooks.projectcalico.org` and abbreviations such as
+`securityeventwebhook.p` and `securityeventwebhooks.p`.
+
+## Sample YAML
+
+```yaml
+apiVersion: projectcalico.org/v3
+kind: SecurityEventWebhook
+metadata:
+  name: jira-webhook
+  annotations:
+    webhooks.projectcalico.org/labels: 'Cluster name:Calico Enterprise'
+spec:
+  consumer: Jira
+  state: Enabled
+  query: type=waf
+  config:
+    - name: url
+      value: 'https://your-jira-instance-name.atlassian.net/rest/api/2/issue/'
+    - name: project
+      value: PRJ
+    - name: issueType
+      value: Bug
+    - name: username
+      valueFrom:
+        secretKeyRef:
+          name: jira-secrets
+          key: username
+    - name: apiToken
+      valueFrom:
+        secretKeyRef:
+          name: jira-secrets
+          key: token
+```
+
+## Security Event Webhook definition
+
+### Metadata
+
+| Field | Description                                               | Accepted Values                                     | Schema |
+| ----- | --------------------------------------------------------- | --------------------------------------------------- | ------ |
+| name  | Unique name to describe this resource instance. Required. | Alphanumeric string with optional `.`, `_`, or `-`. | string |
+
+#### Annotations
+
+SecurityEventWebhooks provide an easy way to add arbitrary data to the webhook generated HTTP payload through the metadata annotation.
```

Comment:
```suggestion
Security event webhooks provide an easy way to add arbitrary data to the webhook generated HTTP payload through the metadata annotation.
```
Like this when speaking about the idea. If you're talking about the object, use code font.

Reply:
Corrected.